client/my-sites/checkout/checkout-thank-you/licensing-thank-you-auto-activation.tsx

```diff
@@ -227,7 +227,7 @@ const LicensingActivationThankYou: FC< Props > = ( {
 								: 'licensing-thank-you-auto-activation__product-info'
 						}
 					>
-						{ translate( 'Hello %(username)s,', {
+						{ translate( 'Hello %(username)s!', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.